src/Console/OutdatedCommand.php

```diff
@@ -0,0 +1,85 @@
+<?php
```

Comment:
I like having console specific classes in their own namespace. It's up to you :)